legacyCode/legacyCode.js

```diff
@@ -1,2 +1,81 @@
-console.log("code is there");
+function enterSearchTerm(event) {
+  event.preventDefault();
+  var searchTerm = document.getElementById("searchTerm");
```

Comment:
Save reference to all needed html elements and reuse them (instead of searching DOM every time).